auth/src/swig/auth.i

```diff
@@ -1193,21 +1118,7 @@ static CppInstanceManager<Auth> g_auth_instances;
 
 %typemap(csclassmodifiers) firebase::auth::Auth "public sealed class";
 %typemap(csclassmodifiers) firebase::auth::User "public sealed class";
```

Comment:
I believe `firebase::auth::User` should be `internal sealed class` now since we do not want to expose `FirebaseUserInternal`

Reply:
Ah, it was getting changed to internal by the post process thing that changes things with "Internal" in the name, but it is definitely confusing, so will change it here as well.